tests/core/basic20/task.yaml

```diff
@@ -67,3 +67,6 @@ execute: |
         BRAND_ID="$BRAND_ID\*"
     fi
     snap recovery --unicode=never | MATCH "[0-9]+ +$BRAND_ID +$MODEL +current"
+
+    echo "snap repair does not fail"
+    /usr/lib/snapd/snap-repair run
```

Comment:
the snap-repair spread test needs to be updated too

Reply:
Yeah, the snap-repair spread test is actually the better place, this part here is redundant once main/snap-repair is enabled on core20, I remove it.